windows.ltsc2022/Dockerfile

```diff
@@ -123,4 +123,7 @@ RUN .\update_path.cmd;
 
 # gcloud requires python on path, update_path.cmd adds python to path. This created a .install\.backup folder that's required for rollbacks and takes ~1Gig. 
 RUN gcloud components install gke-gcloud-auth-plugin --quiet; `
-    Remove-Item -Path C:\google-cloud-sdk\.install\.backup -Force -Recurse
\ No newline at end of file
+    Remove-Item -Path C:\google-cloud-sdk\.install\.backup -Force -Recurse
+
+# remove az cli warning - https://github.com/Azure/arm-deploy/issues/173
+RUN az config set bicep.use_binary_from_path=false
```

Comment:
Same comment on this one - probably worth consolidating it up onto line 50, vs a whole new layer.